scripts/docker/Dockerfile

```diff
@@ -8,7 +8,7 @@ LABEL io.parity.image.authors="devops-team@parity.io" \
 	io.parity.image.vendor="Parity Technologies" \
 	io.parity.image.title="parity/substrate" \
 	io.parity.image.description="Substrate: The platform for blockchain innovators." \
-	io.parity.image.source="https://github.com/paritytech/substrate/blob/master/scripts/docker/Dockerfile" \
+	io.parity.image.source="https://github.com/paritytech/substrate/blob/${VCS_REF}/scripts/docker/Dockerfile" \
 	io.parity.image.revision="${VCS_REF}" \
```

Comment:
That's going to be full SHA, not `SHORT_SHA` now, is it ok?

Reply:
yep, that way this link will be vaild

Reply:
I'm not commenting on the link, but the `image.revision` field (whatever that is)

Reply:
it's metadata, you can read it by `docker inspect $imagename`.
Yes, I think it's totally OK.